client/src/main/java/org/apache/rocketmq/client/impl/consumer/RebalancePushImpl.java

```diff
@@ -57,7 +57,8 @@ public void messageQueueChanged(String topic, Set<MessageQueue> mqAll, Set<Messa
          * Fix: inconsistency subscription may lead to consumer miss messages.
          */
         SubscriptionData subscriptionData = this.subscriptionInner.get(topic);
-        long newVersion = System.currentTimeMillis();
```

Comment:
Maybe it could be nanoseconds

Reply:
This is also a good way, but I think using the increment method is simpler. Both have their own advantages.